HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -230,40 +230,57 @@ def self.apply_infiltration(hpxml)
   def self.apply_attics(hpxml)
     return unless hpxml.has_space_type(HPXML::LocationAtticVented)
 
-    vented_attic = nil
+    vented_attics = []
+    default_sla = Airflow.get_default_vented_attic_sla()
+    default_ach = nil
     hpxml.attics.each do |attic|
       next unless attic.attic_type == HPXML::AtticTypeVented
+      # check existing sla and ach
+      default_sla = attic.vented_attic_sla unless attic.vented_attic_sla.nil?
+      default_ach = attic.vented_attic_ach unless attic.vented_attic_ach.nil?
 
-      vented_attic = attic
+      vented_attics << attic
     end
-    if vented_attic.nil?
+    if vented_attics.empty?
       hpxml.attics.add(id: 'VentedAttic',
-                       attic_type: HPXML::AtticTypeVented)
-      vented_attic = hpxml.attics[-1]
+                       attic_type: HPXML::AtticTypeVented,
+                       vented_attic_sla: default_sla)
+      hpxml.attics[-1].vented_attic_sla_isdefaulted = true
     end
-    if vented_attic.vented_attic_sla.nil? && vented_attic.vented_attic_ach.nil?
-      vented_attic.vented_attic_sla = Airflow.get_default_vented_attic_sla()
+    vented_attics.each do |vented_attic|
+      next unless vented_attic.vented_attic_sla.nil? && vented_attic.vented_attic_ach.nil?
+      if not default_ach.nil? # ACH specified
+        vented_attic.vented_attic_ach = default_ach
+      else # Use SLA
+        vented_attic.vented_attic_sla = default_sla
+      end
       vented_attic.vented_attic_sla_isdefaulted = true
```

Comment:
@shorowit If there's ach/sla specified in any attic, then other attics will be defaulted as the same ach/sla. Otherwise we call default sla method. In the first case, do we want the `vented_attic_sla_isdefaulted` to be true for attics except for the one that was explicitly specified?

Reply:
Hmm. I'm wondering if we should simplify these types of questions by changing EPvalidator to allow multiple ventilation rates (one per attic) to be specified, and then perform defaulting on a per-attic basis. Anywhere in our model (airflow.rb, hvac_sizing.rb), we'd simply use a floor-area-weighted average value. What do you think?

Reply:
Sounds fair to me.

Reply:
One question for vented attic, it seems both ach and sla inputs are allowed for vented attic (while only sla was requested in [hvac_sizing](https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/hvac_sizing.rb#L2577-L2578), is this intended? Or is ach only possible in some specific cases?). So if there're multiple vented attics of which some are specifying ventilation rates in ach and some are in sla, and some are not specifying anything (defaulted). What do you think is a better way to average them? One possible approach is to convert them all into either sla or ach form then do averaging.

Reply:
Proceed with allowing one VentilationRate